testing/kokoro/system_tests.sh

```diff
@@ -138,7 +138,7 @@ if [ $GOLANG_SAMPLES_GO_VET ]; then
   diff -u <(echo -n) <(gofmt -d -s .)
 
   # Remove the golang-only triple-dot suffix.
-  target_dir="${TARGET%/...}"
+  target_dir=$(echo "$TARGET" | sed -r 's/\/\.\.\.$//')
```

Comment:
Could we use `TARGET_DIRS`, which doesn't have the `./...` at all?

Reply:
Somehow I missed that TARGET_DIRS could be a space-delimited list of directories. I can use it, but will need to make other modifications.

Reply:
TIL find can take a space-delimited list of directories